CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Contributing
+
+🙏 Thanks for your interest in contributing!
+
+## Setup
+
+After cloning the repo, run the following command to install dependencies:
+
+```sh
+yarn
+```
+
+This project uses [Yarn Workspaces](https://yarnpkg.com/lang/en/docs/workspaces/) to manage multiple packages within one repository. Check out the `workspaces` key within `package.json` to view all possible workspace directories. Some workspaces of note:
+
+- `packages/gatsby-mdx`: the source for the gatsby-mdx plugin
+- `examples/docs`: the gatsby-mdx docs site (built with [Gatsby](https://www.gatsbyjs.org/))
```

Comment:
might want to link to https://next.gatsbyjs.org/ here until the rc releases become stable releases (since we only work with gatsby 2+)

Reply:
Good call, will update!